style.css

```diff
@@ -1 +1,32 @@
+#image {
+  max-width: 100%;
+  height: auto;
+  display: block;
+  margin: 0 auto;
+  margin-bottom: 25px;
+}
+#main {
+  text-align: center;
+}
+.header {
+  background-color: gray;
+  color: white;
+  padding: 16px;
+  font-size: 32px;
+}
+body {
+  margin: 0;
+}
+footer {
+  background-color: lightgray;
+  color: black;
+  padding-inline: 250px;
+  padding-block: 25px;
+  font-size: 22px;
+  text-align: justify;
+}
+#img-caption {
+  font-size: 20px;
+  padding-inline: 150px;
 
```

Comment:
There are problems with your linters. Before adding your file please check the linters with npx hint . and npx stylelint "**/*.{css,scss}" commands.